src/audio/host-legacy.c

```diff
@@ -180,8 +180,11 @@ static uint32_t host_get_copy_bytes_one_shot(struct host_data *hd, struct comp_d
 	copy_bytes = ALIGN_DOWN(copy_bytes, hd->dma_copy_align);
 
 	split_value = host_dma_get_split(hd, copy_bytes);
-	if (split_value)
+	if (split_value) {
+#ifndef CONFIG_DISABLE_DESCRIPTOR_SPLIT
```

Comment:
`if (!IS_ENABLED(CONFIG_DISABLE_DESCRIPTOR_SPLIT) && split_value)` but in fact you don't need the whole `split_value` variable is splitting is disabled

Reply:
Yes, will it modify.